CONTRIBUTING.md

```diff
@@ -0,0 +1,202 @@
+# Contributing to D2iQ docs
+
+## Add/Edit content to existing pages
+
+### TL;DR Common Gotchas
+
+1.  The docs site uses metalsmith markdown rendering, which is similar to but not exactly github flavored markdown (gfm).
+1.  Every folder must have one and only one corresponding index.md file, naming is limited as the folder name becomes part of the link structure, choose wisely.
+1.  Every index.md file must have the minimum front matter metadata of:
+    layout, title, navigationTitle, menuWeight and excerpt.
+
+        ![docs metadata](https://i.imgur.com/8KPsvEY.png)
+
+1.  Code blocks within numbered lists need to be indented by 4 spaces
+1.  Relative links after a lot of edits or restructuring need to be checked that they are catching the right folder structure, please be careful.
+1.  Make sure to switch to all html tags within html code blocks for notes and warnings.
```

Comment:
I think it's good to call out menu weight but I don't think this TLDR section should get too heavy. There is already specific info about menuWeight further down the page.

That said, the point of this PR is to organize the docs rather than to define their content too granularly, so maybe we open a new PR with this change and discuss it therE?

Reply:
Agree on the TLDR. This should be discussed elsewhere as a MUST in the front matter.